src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -97,6 +98,38 @@ const GovernanceReviewTeam = () => {
               </div>
             </dl>
           </div>
+
+          <div
+            className={classnames({
+              'bg-base-lightest': closedIntakeStatuses.includes(
+                systemIntake.status
+              ),
+              'easi-grt__status--open': openIntakeStatuses.includes(
+                systemIntake.status
+              )
+            })}
+          >
+            <div className="grid-container overflow-auto">
+              <dl className="easi-grt__status-info text-gray-90">
+                <dt className="text-bold">{t('status.label')}</dt>
+                &nbsp;
+                <dd
+                  className="text-uppercase text-white bg-base-dark padding-05 font-body-3xs"
+                  data-testid="grt-status"
+                >
+                  {closedIntakeStatuses.includes(systemIntake.status)
```

Comment:
I would maybe export this check to a util function somewhere. Partially to make the code more readable here, but also because if we export this bit to a util function, then we can unit test it with all 50 of our statuses, and the code will be a bit more robust.